### ISSUE TITLE
Please Update Conda Install

### ISSUE THREAD

**Issue by WilliamJudge94** [tensorflow/tensorflow#42568]
<em>Please make sure that this is an issue related to performance of TensorFlow.
As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:performance_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- TensorFlow installed from conda then used pip to update tensorflow to 2.3.0):
- TensorFlow version (2.3.0):
- Python version 3.7:
- CUDA/cuDNN version (Latest conda install verison):
- GPU model and memory (V100 32gb VRAM):

**Describe the current behavior**
Tensorflow is unbelievable slow to start the training. Takes 30-40 min to initialize. Stops halfway through the first epoch for 10 minutes. Then every epoch after that is fine. I have run a similar code in Google Colab and it runs perfectly normal. Does not take 30-40 min to initialize. Everything is a fresh install (conda, venv, etc). 

**Describe the expected behavior**
For model.fit() to start training within a minute or two. Not take 40 minutes.

**Standalone code to reproduce the issue**

conda install -c anaconda tensorflow-gpu (I do this because it installs all the cuda drivers in one command. It is impossible to install cuda drivers on a cluster login node that does not contain an nvidia GPU inside of it. Conda is the only way to install TensorFlow with GPU acceleration on computational shared clusters.)

pip install tensorflow-gpu==2.3.0 (because your latest conda version does not install tensorflow-gpu==2.3.0)

Use the ImageDataGenerator:

batch_size=32

train_gen = train_datagen.flow(X_train, y_train, 
                                 batch_size=batch_size, 
                                 shuffle=True,
                                 subset='training', seed=7)

v_gen = val_datagen.flow(X_test, y_test,
                                 batch_size=batch_size,
                                 subset='training', seed=7)

ds = tf.data.Dataset.from_generator(lambda: train_gen, 
    output_types=(tf.float32, tf.float32), 
    output_shapes=([32,64,64,3], [32,3])
)

v = tf.data.Dataset.from_generator(lambda: v_gen, 
    output_types=(tf.float32, tf.float32), 
    output_shapes=([32,64,64,3], [32,3])
)

model.fit_generator(ds,
      epochs=100,  #### set repeat in training dataset
      steps_per_epoch=len(train_gen),
      validation_data=v,
      validation_steps=len(v_gen), 
      callbacks=[checkpoints, ], verbose=1)

You can use model.fit() as well. There is still a 40-minute wait for this to start training.
